src/utils/jwt/jws.ts

```diff
@@ -50,7 +50,7 @@ async function importPrivateKey(key: SignatureKey, alg: KeyImporterAlgorithm): P
     throw new Error('`crypto.subtle.importKey` is undefined. JWT auth middleware requires it.')
   }
   if (isCryptoKey(key)) {
-    if (key.type !== 'private') {
+    if (key.type !== 'private' && key.type !== 'secret') {
       throw new Error(`unexpected non private key: CryptoKey.type is ${key.type}`)
```

Comment:
How about changing the error message to clarify the `secret` is also expected? Like this:

```ts
throw new Error(`unexpected key type: CryptoKey.type is ${key.type}, expected 'private' or 'secret'`)
```

Reply:
thanks for checking it out. updated the comment.